views/backend/dashboard.php

```diff
@@ -82,7 +70,7 @@ class="material-icons">more_vert</i></a>
 
                         <div class="modal" id="comment_<?=$comment->id ?>">
                             <div class="modal-content">
-                                <h4><?= $comment->title ?></h4>
+                                <h4><?= $comment->name ?></h4>
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$comment'.](https://app.codacy.com/manual/JeromeLacquemant/projet-5/pullRequest?prid=5379676)